lib/lsblk/index.js

```diff
@@ -80,11 +80,15 @@ const lsblk = (callback) => {
 
     let devices = null;
 
-    try {
-      devices = SUPPORTS_JSON ? parseJSON(stdout) : parsePairs(stdout);
-    } catch (e) {
-      callback(e);
-      return;
+    if (SUPPORTS_JSON) {
+      try {
+        devices = parseJSON(stdout);
+      } catch (e) {
+        SUPPORTS_JSON = false;
+        devices = parsePairs(stdout);
```

Comment:
This won't work, stdout will be some malformed json here, you can't parse it with `parsePairs`.